src/anm/analytics.js

```diff
@@ -43,6 +37,26 @@ var Analytics = function () {
     this.trackPlayingComplete = this.trackPlayer('playing_complete');
 };
 
+/**
+ * @param {Array} views - an array of views
+ */
+Analytics.prototype.sendData = function sendData(views) {
```

Comment:
I think having two methods with similar names `sendData`/`sendPlayerData` is not very good, either one of them could be private (marked with `@private`) or they should have different names.